.gitignore

```diff
@@ -0,0 +1,5 @@
+.autosave
+moc_*
+weather-qt
+ui_*
+*.user
```

Comment:
I'd add these as well, I forgot to include them in my initial comment

*.o
Makefile